GTM/install.sh

```diff
@@ -86,13 +86,6 @@ fi
 echo "Downloading ydbinstall"
 curl -s -L https://raw.githubusercontent.com/YottaDB/YottaDB/master/sr_unix/ydbinstall.sh -o ydbinstall
 
-# Verify hash as we are going to make it executable
-sha1sum -c --status ydbinstall_SHA1
```

Comment:
I'm torn about removing this - yes it removes an issue when ydbinstall is updated, but if the download fails for whatever reason and downloads an http file and executes it - it won't work. maybe we should grab the first line of the file and make sure it matches a regex?